packages/Search/src/details/DTK_DetailsTreeConstruction_def.hpp

```diff
@@ -202,8 +203,8 @@ class CalculateBoundingBoxesFunctor
     }
 
   private:
-    Kokkos::View<Node *, DeviceType> _leaf_nodes;
-    Node *_root;
+    BoundingVolumeHierarchy<DeviceType> _bvh;
+    Kokkos::View<Node *, DeviceType, Kokkos::MemoryUnmanaged> _leaf_nodes;
```

Comment:
```suggestion
    Kokkos::View<Node const *, DeviceType, Kokkos::MemoryUnmanaged> _leaf_nodes;
```

Reply:
And why did you chose to pass the `MemoryUnmanaged` template argument?

Reply:
Removed.